packages/context/src/inject.ts

```diff
@@ -351,15 +376,22 @@ function resolveAsSetter(ctx: Context, injection: Injection) {
       `The type of ${targetName} (${targetType.name}) is not a Setter function`,
     );
   }
-  const bindingSelector = injection.bindingSelector;
-  if (!isBindingAddress(bindingSelector)) {
+  const key = injection.bindingSelector;
+  if (!isBindingAddress(key)) {
```

Comment:
Can we keep `bindingSelector` please? I find that name more descriptive than just `key`.